executable/microsoft_pe.ksy

```diff
@@ -408,3 +413,144 @@ types:
         size: length - 8
         doc: Contains a certificate, such as an Authenticode signature.
     doc-ref: 'https://docs.microsoft.com/en-us/windows/desktop/debug/pe-format#the-attribute-certificate-table-image-only'
+
+  resource_directory_table:
+    params:
+      - id: depth
+        type: u4
+      - id: rsrc_section_file_offset
+        type: u4
+      - id: rsrc_section_virtual_address
+        type: u4
+    seq:
+      - id: characteristics
+        type: u4
```

Comment:
aren't they bit flags? Shouldn't they be a bit-sized type?

Reply:
They're not currently used and reserved for future use. I've added a comment to specify this.

Reply:
I recommend renaming the field to `reserved` as per the [style guide](https://doc.kaitai.io/ksy_style_guide.html#attr-id). The name `characteristics` really suggests that it's something meaningful.

Reply:
Done